CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 - Upgrade `ArgillaWrapperClient` to use Argilla v2.x
 - (Beta) Add `DataClient` and `StudioDatasetRepository` as connectors to Studio for submitting data.
 - Add the optional argument `generate_highlights` to `MultiChunkQa`, `RetrieverBasedQa` and `SingleChunkQa`. This makes it possible to disable highlighting for performance reasons.
+- `min_score` and `max_results` are now optional parameters in `DocumentIndexClient.SearchQuery`.
+- `k` is now an optional parameter in `DocumentIndexRetriever`.
```

Comment:
these, as well as the breaking change, should go into the `unreleased` category, as 6.0.0 is already released